.gitignore

```diff
@@ -4,11 +4,13 @@
 *.swo
 gae_runtime/*
 third_party/*
+assets/hashes.js
 backend_prod_files/*
 core/templates/prod/*
 core/tests/.browserstack.env
 .coverage*
 .viminfo
+.vscode/*
```

Comment:
Isn't this developer-specific? I think we need not add this to gitignore. Suggestions?

Reply:
I was having a bunch of VSCode config files showing up in my `git status` so I do want to prevent other dev accidentally committing those files as well.

Reply:
OK. Not too sure but let's keep it this way then. Thanks!

Reply:
Just confirming this is fine to add. .gitignore is exactly for things like this (see e.g. .viminfo just above it).